janus_messages/src/lib.rs

```diff
@@ -193,7 +198,7 @@ impl Decode for Time {
     }
 }
 
-/// PPM protocol message representing a half-open interval of time with a resolution of seconds;
+/// DAP protocol message representing a half-open interval of time with a resolution of seconds;
 /// the start of the interval is included while the end of the interval is excluded.
 #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
 pub struct Interval {
```

Comment:
A wrapper type for `janus_messages::Interval` struck me as the easiest way to integrate gracefully with `Row::get` and the stuff for constructing queries, so that's what I did here.